server/requirements.txt

```diff
@@ -1,15 +1,21 @@
+black==20.8b1
 flask==1.1.2
 flask-cors==3.0.8
 flask_restful==0.3.8
 gensim==3.8.2
+gsutil==4.59
 gunicorn==20.0.4
+lxml==4.6.2
 numpy==1.19.5
 pandas==1.2.2
 PyMuPDF==1.18.8
 requests==2.25.1
+rpy2==3.4.2
 scikit-learn==0.22.1
+scipy==1.5.0
 sentry-sdk[flask]==0.17.0
 smart-open==3.0.0
 spacy==3.0.3
 tables==3.6.1
 tensorflow==2.4.1
+tqdm==4.56.2
```

Comment:
let's also include `wheel`, which is needed for installations of a few packages. (I just realized that `python -m venv` doesn't include it by default but `virtualenv` command does.)